components/script/dom/bindings/reflector.rs

```diff
@@ -50,6 +51,18 @@ impl Reflector {
         self.object.handle()
     }
 
+    /// Get the JS runtime.
+    #[inline]
+    pub fn get_runtime(&self) -> *mut JSRuntime {
+        unsafe { JS_GetObjectRuntime(self.object.get()) }
+    }
+
+    /// Get the JS context.
+    #[inline]
+    pub fn get_cx(&self) -> *mut JSContext {
+        unsafe { JS_GetContext(self.get_runtime()) }
+    }
```

Comment:
Why? We might as well use `Runtime::new()`; that's probably faster than two out-of-line calls.